extract/extract.js

```diff
@@ -0,0 +1,24 @@
+var numEvents = numEvents || 1;
+
+function skipNullOrEmptyValue(key, value) {
+    return (
+        value == null ||
+        (typeof value == 'object' && Object.keys(value).length == 0)
+    ) ? undefined : value;
+}
+
+// The original find query used the same regex on the object property.
+// However, many PR events don't have "problemroulette" in that property.
+
+db.event.find({
+    raw: {
+        $regex: /problemroulette/
```

Comment:
As I was finishing the documentation yesterday, I thought of this, too.  Which is why it mentions the feature will be added soon.  Since I'm not alone in thinking of this feature, it may be added sooner than I expected!  😆 

When I have some spare time (waiting for a plane, in between conf. sessions, etc.), I'll add the feature.  It will be implemented in much the same way as the option for the number of events.

One of the reasons I didn't implement other search terms earlier is because I couldn't decide whether the user should be allowed to specify the entire query (i.e., everything within `find(…)`) or just a regex for the `raw` attribute.  For the sake of expedience, I'll implement it as only the regex.  If we find a need to allow a more complex query, we can implement that later.